### PR TITLE
[Models] support  swa for fleet fallback model

### PR DESCRIPTION
**Pull request by xiaoguoguo626807**
## Motivation
Support sliding-window attention (SWA) for PaddleFleet fallback MLA models, so SWA layers selected by `window_attn_skip_freq` can use the DSA attention path.

## Modifications
- `fastdeploy/model_executor/models/paddleformers/base_fleet.py`: pass `window_attn_skip_freq` and `sliding_window` into `FastDeployAttention`.
- `fastdeploy/model_executor/models/paddleformers/base_fleet.py`: add an SWA branch that builds `indexer_top_k` and calls `DSAAttentionBackend.forward_static` for MLA attention.
- `fastdeploy/model_executor/models/paddleformers/base_fleet.py`: adjust PaddleFleet fallback config/model loading and MoE-related config fields.

## Usage or Command
N/A

## Accuracy Tests
N/A（当前 PR 未提供 SWA/fallback logits 对齐或精度测试结果）

## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [ ] Provide accuracy results.
- [x] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

File: fastdeploy/model_executor/models/paddleformers/base_fleet.py [hunk 1]
```diff
@@ -167,85 +171,117 @@ def squeeze_to_3d(t: paddle.Tensor, name: str) -> paddle.Tensor:
...
-                            qkv=None,
-                            compressed_kv=compressed_kv,
-                            k_pe=k_pos_emb,
-                            forward_meta=forward_meta,
+                    k_pos_emb_sq = k_pos_emb.squeeze(0)
+
+                    if self.window_attn_skip_freq is not None and self.window_attn_skip_freq[self.layer_id] == 1:
+                        kv_lora_rank = self.config.kv_lora_rank
+
+                        q_input = squeeze_to_3d(q_absorbed, "q_absorbed") if q_absorbed.ndim == 4 else q_absorbed
```
[the remark this answers is not in the document]
Comment (xiaoguoguo626807) [reply fastdeploy/model_executor/models/paddleformers/base_fleet.py:181]: q_absorbed 不会是None

**Review by chang-wenbin — APPROVED** [review]
